source/CINetworkUtils.cpp

```diff
@@ -11,6 +11,28 @@
 #include "CINetworkUtils.h"
 #include "CILocation.h"
 
+/**
+ * Decodes 12 bytes into a 12 character string.
+ */
+string NetworkUtils::decodeString(uint8_t c1, uint8_t c2, uint8_t c3, uint8_t c4, uint8_t c5, uint8_t c6,
```

Comment:
I don't care too much but it might be better to take in a vector of length 12 here